lighthouse-core/test/audits/seo/tap-targets-test.js

```diff
@@ -10,7 +10,9 @@
 const TapTargetsAudit = require('../../../audits/seo/tap-targets.js');
 const assert = require('assert');
 
-function auditTapTargets(tapTargets, metaElements = [{
+const fakeContext = {computedCache: new Map()};
```

Comment:
doesn't seem like this one needed to become async ;) but I like it 👍

Reply:
Computed artifacts return a promise, so I think tap targets needs to be async to use one?

Reply:
yeah but you're not using `await` in here right?

Reply:
Oh yeah that makes sense.